sapi/litespeed/lsapilib.c

```diff
@@ -1872,10 +1872,12 @@ static char * GetHeaderVar( LSAPI_Request * pReq, const char * name )
 
 char * LSAPI_GetEnv_r( LSAPI_Request * pReq, const char * name )
 {
-    struct LSAPI_key_value_pair * pBegin = pReq->m_pEnvList;
-    struct LSAPI_key_value_pair * pEnd = pBegin + pReq->m_pHeader->m_cntEnv;
     if ( !pReq || !name )
         return NULL;
+
+    struct LSAPI_key_value_pair * pBegin = pReq->m_pEnvList;
```

Comment:
This wouldn't work for standard C compiler - definitions should come before the code.